packages/nextjs/package.json

```diff
@@ -81,11 +81,11 @@
     "@types/node": "^18.19.33",
     "@types/react": "*",
     "@types/react-dom": "*",
-    "next": "^14.2.4",
+    "next": "^14.2.10",
     "typescript": "*"
   },
   "peerDependencies": {
-    "next": "^13.5.4 || ^14.0.3 || >=15.0.0-rc",
+    "next": "^14.2.10 ^14.2.10 >=14.2.10",
```

Comment:
This is definitely incorrect, but we should discuss what we want to support per the old peerDeps.